Community/Tdarr_Plugin_dt0y_DaveThe0nly_HWACCEL_SINGLE_TRANSCODER.js

```diff
@@ -0,0 +1,576 @@
+  : `${Math.round((rate / 1000) * 10) / 10}k`);
+
+// @TODO: Still requires to work on DTS sound and its counterpart to Atmos
+// @TODO: finish other hw acceleration implementations (I only have access to Intel enabled QSV)
+// @TODO: Add option to do HDR -> SDR Tone Mapping (That is not something I want, might take a while)
+const details = () => ({
+  id: 'Tdarr_Plugin_dt0y_DaveThe0nly_HWACCEL_SINGLE_TRANSCODER',
+  Stage: 'Pre-processing',
+  Name: 'OnePassMonsterTranscoder',
+  Type: 'Video',
+  Operation: 'Transcode',
+  // eslint-disable-next-line max-len
+  Description: '[Contains built-in filter] Single-pass transcoder to optimize a media file to be used with Plex/Emby etc.',
+  Version: '1.0',
+  Tags: 'pre-processing,ffmpeg,hevc,x264,vaapi,nvenc,configurable,audi,video,subtitles',
+  Inputs: [
+    {
+      name: 'targetResolutionBitrate',
+      type: 'string',
+      defaultValue: '4KUHD:10M,1080p:5M',
+      inputUI: {
+        type: 'text',
+      },
+      // eslint-disable-next-line max-len
+      tooltip: `A mapping of resolution(${supportedResolutions.join(', ')}):bitrate(FFMPEG friendly format 8M or 8000k),
+      seperated by a comma eg: "4KUHD:10M,1080p:5M,etc...".
+      Keep blank to keep video stream as is (still transcodes to HEVC).`,
+    },
+    {
+      name: 'hwAccelerate',
+      type: 'string',
+      defaultValue: 'none',
+      inputUI: {
+        type: 'dropdown',
+        options: Object.values(HWACCEL_OPTIONS),
+      },
+      tooltip: 'Keep blank if none',
+    },
+    {
+      name: 'forceHevc',
+      type: 'boolean',
+      defaultValue: false,
+      inputUI: {
+        type: 'dropdown',
+        options: [
+          'false',
+          'true',
+        ],
+      },
+      tooltip: 'Forces HEVC conversion on x264 video',
+    },
+    {
+      name: 'hevcCompressionLevel',
+      type: 'number',
+      defaultValue: 7,
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: 'Sets the compression level for HEVC transcoding (1-7)',
+    },
+    {
+      name: 'x264quality',
+      type: 'string',
+      defaultValue: 'veryfast:27',
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: 'Sets quality of x264 being preset:crf (veryfast:27) being default',
+    },
+    {
+      name: 'createOptimizedAudioTrack',
+      type: 'string',
+      defaultValue: 'ac3:6,aac:6,aac:3',
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: 'Creates an optimized audio track/s codec:channels (aac:2,ac3:6)',
+    },
+    {
+      name: 'keepAudioTracks',
+      type: 'string',
+      defaultValue: 'eng',
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: 'Keeps an audio track of selected language/s, separated by ",", if empty all are kept',
+    },
+    {
+      name: 'outputContainer',
+      type: 'string',
+      defaultValue: '.mp4',
+      inputUI: {
+        type: 'dropdown',
+        options: ['.mp4', '.mkv', 'keep'],
```

Comment:
Please keep all options on separate lines like you did for the true/false dropdown :D